src/sprout/Helpers/AI/AI.php

```diff
@@ -0,0 +1,63 @@
+<?php
+namespace Sprout\Helpers\AI;
+
+use Exception;
+use Kohana;
+use Sprout\Helpers\AI\AiApiBase;
+use Sprout\Helpers\AI\OpenAiApi;
+
+
+/**
+ * General helpers for AI system
+ */
+class AI
+{
+
+    /**
+     * Available classes for AI.
+     *
+     * Core config ai.enabled_class should be one of these values
+     */
+    const AI_CLASSES = [
+        OpenAiApi::class => 'OpenAi',
+    ];
+
+
+    /**
+     * Check if AI usage is enabled (class configured and return enabled)?
+     *
+     * @return bool
+     */
+    public static function isEnabled(): bool
+    {
+        $enabled_class = Kohana::config('ai.enabled_class');
+        $class = array_search($enabled_class, self::AI_CLASSES);
+
+        if ($class === false) {
+            throw new Exception('Invalid AI class: ' . $enabled_class);
+        }
+
+        /** @var AiApiBase */
+        $class = new $class();
```

Comment:
Another possible spot for `Sprout::instance()`.

I did have a branch somewhere where I added generic typings to that helper. Meaning you don't have to assert the type with a comment and the class type flowed in from the class check.